website/pages/_app.tsx

```diff
@@ -1,17 +1,21 @@
 import '../styles/globals.scss';
 import type { AppProps } from 'next/app';
 import { Layout } from '../components/Layout';
-import colors from '../styles/variables.module.scss';
-import { UserProvider } from '@auth0/nextjs-auth0';
+
+type Props = {
+  children?: React.ReactNode;
+};
+
+const AppNotLogged: React.FC<Props> = ({ children }) => {
+  return <Layout>{children}</Layout>;
+};
+
+const AppLogged: React.FC<Props> = ({ children }) => {
+  return <Layout>{children}</Layout>;
+};
```

Comment:
What's the difference between these 2 here? Is it for later?

Reply:
it's for later to delete